index.js

```diff
@@ -12,6 +12,8 @@ const url = require("url");
 // @ts-ignore https://github.com/peterbe/minimalcss/pull/30
 const minimalcss = require("minimalcss");
 
+const { homepage } = require(`${process.cwd()}/package.json`);
```

Comment:
code related to `homepage` resolution should be moved to `run.js`

Reply:
ok 👍